addon/components/paper-button.js

```diff
@@ -4,45 +4,53 @@ import RippleMixin from '../mixins/ripple-mixin';
 import ProxiableMixin from 'ember-paper/mixins/proxiable-mixin';
 import ColorMixin from 'ember-paper/mixins/color-mixin';
 
+const {
+  assert,
+  computed,
+  isPresent
+} = Ember;
+
 export default BaseFocusable.extend(RippleMixin, ProxiableMixin, ColorMixin, {
   attributeBindings: ['target', 'action', 'type'],
   tagName: 'button',
   themed: true,
-  classNameBindings: ['raised:md-raised', 'icon-button:md-icon-button', 'focus:md-focused', 'themed:md-default-theme', 'themed:md-button'],
+  classNameBindings: [
+    'raised:md-raised',
+    'icon-button:md-icon-button',
+    'focus:md-focused',
```

Comment:
this is pointless. BaseFocusable already has such a binding: https://github.com/miguelcobain/ember-paper/blob/wip/v1.0.0/addon%2Fcomponents%2Fbase-focusable.js#L14

Also, `focus` was renamed to `focused`. Anyway, this logic is no longer handled in paper-button.